assets/src/components/propertiesPanel/miniMap.tsx

```diff
@@ -53,25 +55,26 @@ const MiniMap = ({
 }) => {
   const stations: Stop[] | null = useStations()
 
-  return inTestGroup(MAP_BETA_GROUP_NAME) ? (
+  const inMapBetaGroup = inTestGroup(MAP_BETA_GROUP_NAME)
+
+  return (
     <MapFollowingPrimaryVehicles
       selectedVehicleId={vehicle.id}
       vehicles={[vehicle]}
       shapes={shapes}
       secondaryVehicles={routeVehicles}
       stations={stations}
-      allowFullscreen={false}
+      allowFullscreen={!inMapBetaGroup}
```

Comment:
**question:** this restores the ability of non-beta group users to fullscreen the minimap, is that intentional?

Reply:
Oh, wait, I think I just read the code wrong, this is fine.